wordpress-mobile/WordPress-iOS
# ULS Get Started: email handling

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/404
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/435
Kit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/278

This uses the Kit and Auth changes to process an entered email address.

Notes:
- You may want to test on a real device to ensure magic link emails work.
- All these flows should be fully functional, so you should be able to login and signup as expected.

To test:

---
- Enable the `unifiedWordPress` feature flag.
- On the prologue, `Continue with WordPress.com`.
- The Get Started view appears.

![get_started](https://user-images.githubusercontent.com/1816888/92050110-7932f680-ed49-11ea-8f67-61742737ad0c.jpg)

---
Passwordless WP account:
- Enter an email that has a WP account with no password.
- Verify you receive a login link email, and the `Check your email...` view is displayed.

![open_mail](https://user-images.githubusercontent.com/1816888/92050139-8bad3000-ed49-11ea-821e-cf628bad890d.jpg)

---
WP account with password:
- Enter an email that has a WP account with a password set.
- Verify the Password view is displayed.

![password](https://user-images.githubusercontent.com/1816888/92050149-910a7a80-ed49-11ea-89a5-4b9f53590356.jpg)

---
No matching WP account:
- Enter an email that has not been used on a WP account.
- Verify the Sign Up view is displayed.

![sign_up](https://user-images.githubusercontent.com/1816888/92050160-9667c500-ed49-11ea-80d0-b1f4609c0166.jpg)

---
PR submission checklist:
